packages/clerk-js/src/ui/userProfile/account/profileCard/ProfilePhoneNumbers.tsx

```diff
@@ -24,12 +24,10 @@ export function ProfilePhoneNumbers(): JSX.Element {
             <div key={phoneIdent.id}>
               <PhoneViewer phoneNumber={phoneIdent.phoneNumber} />
               {user.isPrimaryIdentification(phoneIdent) && <Tag color='primary'>Primary</Tag>}
-              {phoneIdent.verification.status && (
-                <VerificationStatusTag
-                  className='cl-tag'
-                  status={phoneIdent.verification.status}
-                />
-              )}
+              <VerificationStatusTag
```

Comment:
🙃 
As I can see the `status` attribute is not typed, can we type it so we avoid issues with arbitrary strings ?

Reply:
Changed to `VerificationStatus`